lib/u3d/installer.rb

```diff
@@ -170,7 +172,7 @@ def self.create
                   end
       if UI.interactive?
         unclean = []
-        installer.installed.each { |unity| unclean << unity unless unity.path =~ UNITY_DIR_CHECK }
+        installer.installed.each { |unity| unclean << unity unless clean_install?(unity.path) }
```

Comment:
I would prefer this to be implemented as `unity.clean_install?` I think and keep each class knowledgeable about it.